api/atproto/syncsubscribeRepos.go

```diff
@@ -11,7 +11,7 @@ func init() {
 
 type SyncSubscribeRepos_Commit struct {
 	Blobs  []util.LexLink               `json:"blobs" cborgen:"blobs"`
-	Blocks util.LexBytes                `json:"blocks" cborgen:"blocks"`
+	Blocks util.LexBytes                `json:"blocks,omitempty" cborgen:"blocks"`
```

Comment:
this is a manual hack, and would get clobbered by running `lexgen`, I assume? Ok for now, making lexgen "clean" is a goal but will be a separate pass of work.

Reply:
Yeah, lexgen needs work. I guess i can add a TODO comment here to help us not miss it in future diffs